licenses/LICENSE_mpg123.txt

```diff
@@ -0,0 +1,22 @@
+GNU says that the license terms go into a file called COPYING.

```

Comment:
same as openal. Mpg123 use LGPL2.1 as it's mentioned at mpg123.h
```c
/*
	libmpg123: MPEG Audio Decoder library (version 1.29.0)

	copyright 1995-2015 by the mpg123 project
	free software under the terms of the LGPL 2.1
	see COPYING and AUTHORS files in distribution or http://mpg123.org
*
```
it's better to move license here